DEVELOPMENT.md

```diff
@@ -21,23 +21,14 @@ sudo apt install \
   gstreamer1.0-plugins-ugly \
   libgirepository1.0-dev \
   libgstreamer1.0-0 \
-  libgtk-3-dev \
+  libgtk-4-dev \
+  libadwaita-1-dev
   pip \
   python-gi-cairo \
   python3-gst-1.0 \
   python3-venv
-
-sudo add-apt-repository ppa:apandada1/libhandy-1
-sudo apt update
-sudo apt install libhandy-1-0 libhandy-1-dev
 ```
```

Comment:
Note: I didn't check whether the updated packages work on Ubuntu, as I'm on Fedora, but they should.

Reply:
The packages should be fine. One backslash is gone missing after `libadwaita-1-dev`, could you fix that one?

Reply:
Good catch, thanks!